modules/validations-before-agent-iso-creation.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * installing/installing-with-agent/installing-with-agent.adoc
+
+:_content-type: REFERENCE
+[id="validations-before-agent-iso-creation_{context}"]
+= Validation checks before agent.iso creation
+
+The Agent-based installer performs validation checks on user defined yaml files before the ISO is created. Once the validations are sucessfull, the `agent.iso`
+is created.
+
+.`install-config.yaml`
+
+* `baremetal`, `vSphere` and `none` platforms are supported.
+* If `none` is used as a platform, the number of control planes should be `1` and the total number of worker replicas should be `0`.
+* `apiVIPs` and `ingressVIPs` must be set for bare metal and vSphere platforms respectively.
+
+.`agent-config.yaml`
+
+* Mac address has to be defined for each interface.
+* World Wide Name(WWN) extensions are not supported in root device hints.
+* (WWN) vendor extensions are not supported in root device hints.
+* The `role` parameter in the `host` object must have a value of either `master` or `worker`.
+
+== ZTP manifests
+
+.`agent-cluster-install.yaml`
+
+* For the `networkType` parameter, the supported value is `cidr` which is used for IPv6. OpenShift SDN is not supported.
+
+.`cluster-image-set.yaml`
+
+* The `releaseImage` parameter must have a value that matches the content in the `registries.conf` file.
+
+** `registries.conf` is generated: The `ImageContentSource` parameter in the `install-config.yaml` file must have a value that matches the `releaseImage` parameter value of `mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/latest`.
+** User defined `registries.conf`: `registries.conf` file must have an entry matching the `releaseImage` parameter value of `mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/latest`.
+
+.`nmstateconfig.yaml`
+
+* Static network configuration is not valid.
```

Comment:
Should remove this sentence as nmstateconfig.yaml defines static network configuration. It must be valid nmstate configuration per the examples in https://nmstate.io/examples.html